### ISSUE TITLE
 Error while loading shared libraries (Ubuntu)

### ISSUE THREAD

**Issue by xsahil03x** [putraxor/flutter_desktop_launcher#5]
./flutter_launcher: error while loading shared libraries: libglfw.so.3: cannot open shared object file: No such file or directory

**Comment by putraxor** [comment]
Install `$ sudo apt-get install libglfw3-dev libepoxy-dev libjsoncpp-dev libgtk-3-dev libx11-dev`

**Comment by xsahil03x** [comment]
Thanks @putraxor it worked but i am unable to see any kind of text and most of the images.
![screenshot from 2019-01-10 12-59-46](https://user-images.githubusercontent.com/25670178/50952971-fd32cd80-14d7-11e9-8472-3f46d662dd33.png)

**Comment by putraxor** [comment]
Yes I still encounter with the issue, it possibly related to linux embedder itself

**Comment by xsahil03x** [comment]
Sure and great work.

**Comment by kindlychung** [comment]
Got this:

```
sudo apt-get install libglfw3-dev libepoxy-dev libjsoncpp-dev libgtk-3-dev libx11-dev
```

**Comment by hamidabdulmalik** [comment]
Where are we with this one @putraxor, still can't see any other texts on linux mint.

**Comment by zabilal** [comment]
what a beautiful effort. Am on deepin linux and am encountering 2 the same issue as @xsahil03x . I cant see any text and the HOT Reload is also not working. please if you can attend to this

**Comment by thmaph** [comment]
I have not got to checking hot reload on linux but i have got text working in linux mint, I think this will work for all linux distros.

So far the best solution for text on linux is to use a custom font following this:

[https://flutter.io/docs/cookbook/design/fonts](url)

I followed steps 1 to 3

1. Import the font files
2. Declare the font in the pubspec.yaml
3. Set a font as the default

You can still use the material (flutter) default Roboto font as a custom font.
